h2o-py/h2o/explanation/_explain.py

```diff
@@ -1413,6 +1460,11 @@ def _get_response(mean_response, show_logodds):
         return np.log(mean_response / (1 - mean_response))
     else:
         return mean_response
+def _isnan(value):
```

Comment:
[PEP-8](https://www.python.org/dev/peps/pep-0008/#blank-lines) suggests newlines before the function.
```suggestion


def _isnan(value):
```